integration_tests/dbt_project.yml

```
@@ -6,38 +6,29 @@ version: '0.1.0'
 profile: 'integration_tests' 
 
```

The PostgreSQL error we are seeing seems to be restricted to the work we are doing here. I actually found [this dbt slack post](https://getdbt.slack.com/archives/C2JRRQDTL/p1641427216111500) from @fivetran-jamie back in January detailing this very issue. 

It seems just explicitly defining the schema in the `dbt_project.yml` does the trick. I am not sure why though 🤔. This worked for me locally and was able to re run the package 4 times over without the `relation already exists` error.

```suggestion

models:
  +schema: apple_search_source

```

---

This didn't work :( I had no problems running the integration tests locally (before this fix) actually. For some reason it's only on circle and only with this package (and not the transforms).

---

It looks like the issue was due to us [cascade dropping](https://github.com/dbt-labs/dbt-core/issues/4030) the schema. It seems dbt doesn't know about this and thus throws errors our way. I just created a new schema and didn't drop cascade and it seems to be working now on my [review branch])(https://github.com/fivetran/dbt_apple_search_ads_source/tree/review/jm-notes). 

Can you make the same changes (or just merge my branch into yours) and see if we are good to go!

---

How strange 🤔 Thanks for the tips, I'm not sure why changing the schema names originally didn't work lol maybe it's because we did that AND added the `+schema` this time 😆